docker-compose.yml

```diff
@@ -6,7 +6,7 @@ volumes:
 
 services:
   db:
-    image: mysql
+    image: "mysql:5.7"
```

Comment:
I pinned this version of mysql to what gavel-ci is using. The newer version of mysql is noisy about `utf8` - I think there were some changes and the charset needs to be more explicit about what version of utf-8.